README.md

```diff
@@ -1,2 +1,137 @@
-# Hello-Microverse
-First Module exercise
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**[Hello Microverse]** is a project to master best practises in coding
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Main</summary>
+  <ul>
+    <li><a href="https://html.com/">Html</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **A .html file that includes a header with the text "Hello Microverse!"**
+- **A .css file that includes one class with styles for your header**
+- **A test.md file with the text "This file should be ignored by git".**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/Eramosh/Hello-Microverse.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] As suggested [here](https://github.com/Eramosh/Hello-Microverse/pull/1#discussion_r1212641413), please make sure to delete this section if you decide not to deploy your project. You can add it later whenever you have a live link 🤓⛏